{"name": "test_login_with_invalid_email_and_valid_password", "status": "passed", "start": 1684490408764, "stop": 1684490410228, "uuid": "197054b2-8f37-4beb-8b6c-51497afd0f9c", "historyId": "9d7c15636f7b48d3beb276131c85cb96", "testCaseId": "9d7c15636f7b48d3beb276131c85cb96", "fullName": "tests.test_Login.TestLogin#test_login_with_invalid_email_and_valid_password", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('setup_and_teardown', 'log_on_failure')"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_Login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-VO28CS3"}, {"name": "thread", "value": "22428-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_Login"}]}